### ISSUE TITLE
document object continuously growing with 'deleted' items while editing 

### ISSUE THREAD

**Issue by philsax59** [yjs/yjs#493]
> Please save me some time and use the following template. In 90% of all issues I can't reproduce the problem because I don't know what exactly you are doing, in which environment, or which y-* version is responsible. Just use the following template even if you think the problem is obvious.

**Checklist**

* [ ] Are you reporting a bug? Use github issues for bug reports and feature requests. For general questions, please use https://discuss.yjs.dev/
* [ ] Try to report your issue in the correct repository. Yjs consists of many modules. When in doubt, report it to https://github.com/yjs/yjs/issues/

**Describe the bug**

**To Reproduce**
3. Open console

**Expected behavior**

**Screenshots**

**Environment Information**
 - Browser / Node.js [e.g. Chrome, Firefox, Node.js]
 - Yjs version and the versions of the y-* modules you are using [e.g. yjs v13.0.1, y-webrtc v1.2.1]. Use `npm ls yjs` to find out the exact version you are using.

**Additional context**

**Comment by philsax59** [comment]
sharing a document with Yjs  in a client-server environment I have noticed a document size grow over time, and inspecting the document object I see many 'deleted' item objects that was created during editing sessions on client-side.   this continuous document size increment, drive into lack of memory on server-side and even  does not allow to properly configure the server memory size.   How can I overcome this issue? 
I have tried to review my code refreshing the document on server-side, removing the deleted items,  but that way I fell into sync issues because I did this creating a new doc, with new id,  and the client editing did not result in any update anymore. 
I am quite new to the library and there may be some other shortcuts to avoid document memory size increment. 
reading all the issues I did not find any useful help for this memory problem,  any help or suggestion will be appreciated
